default-templates/new-account/index.html

```diff
@@ -42,6 +42,10 @@ <h1>Data</h1>
         <span class="lead">Public Folder</span>
         <span class="badge">public</span>
       </a>
+      <a href="/private/" target="_blank" class="list-group-item">
```

Comment:
Is it necessary with `target="_blank"`? It's usually an annoyance for most use cases.

Reply:
I don't see a "Good reason" (ref https://css-tricks.com/use-target_blank/) for doing this, but maybe there's something I'm missing?

Reply:
Copy/paste from what was above.

But a good reason would be that the click brings them to Databrowser, which might be very overwhelming at first. They can just close and be back on the homepage.

Reply:
Ok, let's continue that practice for now then